### PR TITLE
chore: bump the resources on react tests

### PR DESCRIPTION
**Pull request by hamidzr**
## Description

react unit tests are hitting the memory limit ceiling and that seems to be affecting the tester. 
https://app.circleci.com/pipelines/github/determined-ai/determined/23907/workflows/efa84a30-61bf-4454-993b-23ed9e9de25b/jobs/770678/resources

bumping the resources seems to help

## Commentary (optional)

https://determined-ai.slack.com/archives/CKQ3EV0G6/p1656609733904379

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by hamidzr** [comment]
I'll rerun the unit tests a couple of times

I saw `EditableMetadata handles metadata removal` failure which ive seen happyn locally as well

File: .circleci/config.yml [hunk 1]
```diff
@@ -1243,7 +1243,7 @@ jobs:
       - image: cimg/node:16.14.2
         environment:
           CI: "true"
-    resource_class: large
+    resource_class: xlarge
```
Comment (hkang1) [inline .circleci/config.yml:1246]: 🙏

Comment (EmilyBonar) [reply .circleci/config.yml:1246]: I upgraded these tests from medium to large a few months ago, hopefully we don't keep having to do this!

Comment (hamidzr) [reply .circleci/config.yml:1246]: I think we can mess with jest config to limit it in how many tests it runs concurrently. but if this is the only issue it's a cheap fix :)

**Review by loksonarius — APPROVED** [review]

**Pull request merged**